rviz_visual_testing_framework/include/rviz_visual_testing_framework/visual_test_fixture.hpp

```diff
@@ -46,14 +46,23 @@
 #include "rviz_visual_testing_framework/internal/visual_test.hpp"
 #include "rviz_visual_testing_framework/page_objects/page_object_with_window.hpp"
 
+#include <ament_index_cpp/get_package_share_directory.hpp>
+#include <ament_index_cpp/get_package_prefix.hpp>
+
 class VisualTestFixture : public testing::Test
 {
 public:
   VisualTestFixture()
   {
     test_name_ = ::testing::UnitTest::GetInstance()->current_test_info()->name();
     executor_ = std::make_shared<Executor>();
-    src_directory_path_ = std::string(_SRC_DIR_PATH);
+    //src_directory_path_ = std::string(_SRC_DIR_PATH);
+    std::string ROS_PACKAGE_NAME = "rviz_visual_testing_framework";
+    try {
+      src_directory_path_ = ament_index_cpp::get_package_share_directory(ROS_PACKAGE_NAME);
```

Comment:
This is no longer an accurate variable name, since it won't point to the source directory, but instead to the package's install directory.

Reply:
Noted. I will leave this uncommented since its probably out of scope of this PR and restrict changes to the test case in cpp. I feel it is also not a good idea to run this try catch in a constructor and hence the results of ```ament_index_cpp::get_package_share_directory``` are always empty here